src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,25 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  max-width: 944px;
+  padding: 50px 40px;
+
+  @media (min-width: 1200px) {
+    padding: 50px 128px;
+  }
```

Comment:
I think this is redundant, your content have max-width and is centralized so there is no need to increase paddings

Reply:
I can't pass the test without this